src/Addon/AddonServiceProvider.php

```diff
@@ -0,0 +1,76 @@
+<?php
+namespace GiveAddon\Addon;
+
+use Give\Helpers\Hooks;
+use Give\ServiceProviders\ServiceProvider;
+
+use GiveAddon\Addon\Helpers\License;
+use GiveAddon\Addon\Helpers\SettingsPage;
+use GiveAddon\Addon\Helpers\Language;
+use GiveAddon\Addon\SettingsPage as AddonSettingsPage;
+use GiveAddon\Addon\Helpers\ActivationBanner;
+
+/**
+ * Example of a service provider responsible for add-on initialization.
+ *
+ * @package     GiveAddon\Addon
+ * @copyright   Copyright (c) 2020, GiveWP
+ */
+class AddonServiceProvider implements ServiceProvider {
+
+	/**
+	 * @inheritDoc
+	 */
+	public function register() {
+		give()->singleton( Activation::class );
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function boot() {
+		// Load add-on translations.
+		Hooks::addAction( 'admin_init', Language::class, 'load' );
```

Comment:
Do we want to load translation only for the backend?

Reply:
I'll fix that.